src/libs/actions/Session.js

```diff
@@ -224,7 +225,7 @@ function restartSignin() {
  * @param {String} accountID
  */
 function setPassword(password, validateCode, accountID) {
-    Onyx.merge(ONYXKEYS.ACCOUNT, {error: '', loading: true});
+    Onyx.merge(ONYXKEYS.ACCOUNT, {...CONST.DEFAULT_ACCOUNT_DATA, ...{loading: true}});
```

Comment:
I think these can all be a little more simple:
```suggestion
    Onyx.merge(ONYXKEYS.ACCOUNT, {...CONST.DEFAULT_ACCOUNT_DATA, loading: true});
```

I tried it in the console and it looks like it works:
![image](https://user-images.githubusercontent.com/1228807/114465046-ce6a9c80-9ba3-11eb-997b-aacd9fe84794.png)